README.md

```
@@ -14,50 +14,34 @@ _Organize ideas and collaborate using Markdown, a lightweight language for text
 
-<!-- For start course, run in JavaScript:
-'https://github.com/new?' + new URLSearchParams({
-  template_owner: 'skills',
-  template_name: 'communicate-using-markdown',
-  owner: '@me',
-  name: 'skills-communicate-using-markdown',
-  description: 'My clone repository',
-  visibility: 'public',
-}).toString()
--->
+<img alt="Image of Yaktocat" src=https://octodex.github.com/images/yaktocat.png width=400>
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)